TaskHouseUnitTests/PasswordSeriveUnitTest.cs

```diff
@@ -0,0 +1,59 @@
+using TaskHouseApi.Service;
+using Xunit;
+
+namespace TaskHouseUnitTests
+{
+    public class PasswordServiceUnitTest
+    {
+        private IPasswordService passwordService;
+
+        public PasswordServiceUnitTest()
+        {
+            passwordService = new PasswordService();
+        }
+
+        [Fact]
+        public void PasswordService_GenerateSaltedHashedPassword_ReturnCorrectSaltAndHashedPassword_WhenGivenVaildInput()
+        {
+            string password = "1234";
+            string salt = "upYKQSsrlub5JAID61/6pA==";
+
+            var result = passwordService.GenerateSaltedHashedPassword(password, salt);
+
+            Assert.Equal("+z490sXHo5u0qsSaxbBqEk9KsJtGqNhD8I8mVBdDJls=", result);
+        }
+
+        [Fact]
+        public void PasswordService_GenerateSaltedHashedPassword_ReturnANotMatchingSaltAndHashedPassword_WhenGivenInvaildPassword()
+        {
+            string password = "12345";
+            string salt = "upYKQSsrlub5JAID61/6pA==";
+
+            var result = passwordService.GenerateSaltedHashedPassword(password, salt);
+
+            Assert.NotEqual("+z490sXHo5u0qsSaxbBqEk9KsJtGqNhD8I8mVBdDJls=", result);
+        }
+
+        [Fact]
+        public void PasswordService_GenerateSaltedHashedPassword_ReturnANotMatchingSaltAndHashedPassword_WhenGivenInvaildSalt()
+        {
+            string password = "1234";
+            string salt = "TupYKQSsrlub5JAID61/6pA==";
+
+            var result = passwordService.GenerateSaltedHashedPassword(password, salt);
+
+            Assert.NotEqual("+z490sXHo5u0qsSaxbBqEk9KsJtGqNhD8I8mVBdDJls=", result);
```

Comment:
what even are these long-ass strings use in your asserts? 0__0 Could they perhaps get a well--named var, or a comment that explains them? Just in case someone else wuld have to maintain the test sometime